<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelImg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>50</number>
    </property>
   </widget>
   <widget class="QComboBox" name="depComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>321</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <item>
     <property name="text">
      <string>Выберите направление</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эндокринология</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кардиология</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelWelcom">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Добро пожаловать&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Выберите отделение&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Дальше</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QComboBox" name="docName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="docLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Укажите врача</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите дату</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="patientName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="patientLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Укажите ваши ФИО</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="result">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="birthdaLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата вашего рождения</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="birthdayEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="phoneLabel">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>290</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ваш телефон</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>labelWelcom</zorder>
   <zorder>result</zorder>
   <zorder>labelImg</zorder>
   <zorder>depComboBox</zorder>
   <zorder>nextButton</zorder>
   <zorder>backButton</zorder>
   <zorder>docName</zorder>
   <zorder>docLabel</zorder>
   <zorder>patientName</zorder>
   <zorder>patientLabel</zorder>
   <zorder>saveButton</zorder>
   <zorder>dateLabel</zorder>
   <zorder>dateEdit</zorder>
   <zorder>birthdaLabel</zorder>
   <zorder>birthdayEdit</zorder>
   <zorder>phoneLabel</zorder>
   <zorder>phoneEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>depComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
